lua/weapons/remotecontroller.lua

```diff
@@ -41,40 +41,55 @@ function SWEP:Holster()
 	if (self.Linked) then
 		self:Off()
 	end
+	
 	return true
 end
 
 function SWEP:OnDrop()
-	if (self.Linked) then
-		self:Off()
-		self.Linked = nil
-	end
+	if (!self.Linked) then return end
```

Comment:
Wiremod mostly uses () in if statement. So i don't really want to see over9000 different coding styles in one project. Also, i was thought that you don't like thousand of commits, so i make it in one commit